### ISSUE TITLE
Any way to change the colors of and fine tune size of  min close max buttons?

### ISSUE THREAD

**Issue by grav3yardshawty** [kupiqu/SierraBreezeEnhanced#139]
its a very small and arbitrary feature probably only relevant to me and not worth the time, but is there any way at all to achieve this? via the editing of files or something else?

**Comment by kupiqu** [comment]
These properties are currently hard coded. Settings could be added but it would need some work.

Also, there are many settings, if more settings are added, they would need some reconfiguration, adding part of them to a new tab.

Unfortunately, I don't have much time to work on SBE at the moment. If you'd like to take a look and add these features, it would be great! If so, please submit a PR.

**Comment by wxmup** [comment]
> I made a fork hard coding the colors I wanted. It could be useful for anyone trying to do the same with their own colors or developing the settings for this. https:github.com/mateushonorato/SweetBreezeEnhanced

Hello, can your branch modify the font color of the window title?

**Comment by mateushonorato** [comment]
> > I made a fork hard coding the colors I wanted. It could be useful for anyone trying to do the same with their own colors or developing the settings for this. https:github.com/mateushonorato/SweetBreezeEnhanced
> 
> Hello, can your branch modify the font color of the window title?

no, but I believe you can do that using Plasma's built-in color scheme feature

**Comment by wxmup** [comment]
> > > I made a fork hard coding the colors I wanted. It could be useful for anyone trying to do the same with their own colors or developing the settings for this. https:github.com/mateushonorato/SweetBreezeEnhanced
> > 
> > 
> > Hello, can your branch modify the font color of the window title?
> 
> no, but I believe you can do that using Plasma's built-in color scheme feature

Sorry, I tried all the font settings related to black in the color, but it didn't work for the title color. Am I looking for the wrong color?

![Image](https://github.com/user-attachments/assets/ca5aecd7-2886-4406-bc2d-73b567ca04bc)

![Image](https://github.com/user-attachments/assets/001128b0-cf1d-400f-819d-b919211900c2)

**Comment by mateushonorato** [comment]
Just tested here, it seems that the Sierra Breeze Enhanced themes doesn't use the colors provided by the Plasma Color Scheme
